JS simple function/index.js

```diff
@@ -1,34 +1,10 @@
 export function func(s, a, b) {
+  if (a.length > 1 || (b != undefined && b.length > 1)) {
```

Comment:
Стоит добавить проверку на пустую исходную строку `s`.

```suggestion
  if (s === "" || a.length > 1 || b.length > 1) {
```

Reply:
Почему мы проверяем только `b` на `undefined`?

Стоит ли `a` также проверять как `b`?